src/sentry/static/sentry/app/routes.tsx

```diff
@@ -22,12 +31,26 @@ import RouteNotFound from 'app/views/routeNotFound';
 import SettingsProjectProvider from 'app/views/settings/components/settingsProjectProvider';
 import SettingsWrapper from 'app/views/settings/components/settingsWrapper';
 
-function appendTrailingSlash(nextState, replace) {
+const appendTrailingSlash: EnterHook = (nextState, replace) => {
   const lastChar = nextState.location.pathname.slice(-1);
   if (lastChar !== '/') {
     replace(nextState.location.pathname + '/');
   }
-}
+};
+
+type CustomProps = {
```

Comment:
These will probably end up being useful somewhere else, not going to export yet though.